doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -382,6 +382,24 @@ local html = import 'html.libsonnet';
             },
           ],
         },
+        {
+          name: 'splitLimitR',
+          params: ['str', 'c', 'maxsplits'],
+          description: [
+            html.p({}, |||
+              As <code>std.splitLimit(str, c, maxsplits)</code> but will split from right to left.
+            |||),
+            html.p({}, |||
+              Note: Versions up to and including 0.18.0 require <code>c</code> to be a single character.
```

Comment:
This doesn't seem relevant as it's a new function. Instead, there should be 'available_since' field present.